### PR TITLE
ReadMe.md updated

### PR DESCRIPTION
**Pull request by KingsleyMcSimon**

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi Kingsley you have made good progress :raised_hands: 

### But there are changes to be made
- [ ] There are no stickler checks on this project, please ensure there is, and it passes  the checks.
- [ ] Use these snap shots as a reference to update your project. These are my observations.
   * The navigation bar is not properly aligned.
   * The links on the footer are also not properly aligned.

The first two images are yours, compare it with the last two from apple.com

Screenshot from 2019-09-07 01-57-32

Screenshot from 2019-09-07 01-58-37


Screenshot from 2019-09-07 01-58-41

Screenshot from 2019-09-07 01-58-41

### Kindly implement these changes and submit another code review form. :muscle:

**Comment by KingsleyMcSimon** [comment]
This is really an old project, I have actually reviewed it as it is expected of me and the changes done can be seen at the Readme branch of the project.

File: index.html [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 <body>
 
+    <main>  
+        
     <nav class="top-nav">
         <ul>
             <li><i class="fa fa-apple" aria-hidden="true"></i></li>
```
Comment (lucasmazo32) [inline index.html:19]: - [ ] The gradient is slightly different. Kindly fix that.

Source | Clone
-- | --
image | image

File: .stickler.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+linters:
```
Comment (lucasmazo32) [inline .stickler.yml:1]: - [ ] Kindly enable the stickler for this repo. You will have to go to https://stickler-ci.com/ to do so.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by KingsleyMcSimon** [comment]
The required changes has been implemented and the project has passed all the stickler test.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :recycle: 
Hi @KingsleyMcSimon, great job so far! There are a few changes to make to complete this project.

- [ ] You still have [HTML validator](https://validator.w3.org/#validate_by_upload) errors. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
- [ ] Your footer appears to be more to the left. Try to center it more, to make it appear like the original site's footer.

__N.B:__ Checkout [Professional Github repository guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for your next projects to keep your repositories professional. Apparently your pull request has no summary. 

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__

**Comment by KingsleyMcSimon** [comment]
Readme Branch:
As required, the html validator check have been passed.
Proper footer styling implemented.
All the stickler error check passed.

**Review by sumancrest0001 — APPROVED** [review]
# Congratulations @KingsleyMcSimon :fire: :clap: :100: 
You ticked all the minimal requirements from the project specifications. The layout looks similar to the original one, and stickler is passing with no HTML validator error. Now, you can merge the branch to master.

**Best of luck for the next project.
Happy coding !!!!**

**Pull request merged**